packages/catalog-process/test/testCreateEService.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { describe, expect, it } from "vitest";
+import { v4 as uuidv4 } from "uuid";
+import { decodeProtobufPayload } from "pagopa-interop-commons-test";
+import { EService, EServiceAddedV2, toEServiceV2 } from "pagopa-interop-models";
+import {
+  eServiceDuplicate,
+  originNotCompliant,
+} from "../src/model/domain/errors.js";
+import {
+  catalogService,
+  eservices,
+  mockEService,
+  postgresDB,
+} from "./catalogService.integration.test.js";
+import {
+  addOneEService,
+  getMockAuthData,
+  readLastEserviceEvent,
+} from "./utils.js";
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const testCreateEService = () =>
```

Comment:
Here you could also avoid using `eslint-disable` by being explicit with the `: void` return type

Reply:
I tried, it's not actually a void

Reply:
Oh right, you actually return the `describe` result, so it could be `ReturnType<typeof describe>` intstead

Reply:
Updated, thanks!